src/main/kotlin/net/ccbluex/liquidbounce/utils/inventory/InventoryManager.kt

```diff
@@ -55,8 +55,19 @@ import kotlin.random.Random
  */
 object InventoryManager : EventListener {
 
+    val isInventoryOpen
+        get() = isInventoryOpenClientSide || isInventoryOpenServerSide
+
+    private val isInventoryOpenClientSide
+        get() = mc.currentScreen is InventoryScreen
+
     var isInventoryOpenServerSide = false
-        internal set
+        internal set(value) {
+            field = value
+            if (!field && value) {
```

Comment:
Isn't this always false? Because you do field = value first, so isn't this basically if (!value && value) ?